app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ActiveRecord::Base
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
```

Comment:
- [ ] Here also you are saying that a post belongs to `author` in the Users table but in your migrations, you have a reference to users table through the `user_id` and not `author_id`. Kindly fix this to something like `belongs_to :user`

Reply:
This has been resolved